packages/synapse-react-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "synapse-react-client",
-  "version": "3.1.23",
+  "version": "3.1.22",
```

Comment:
3.1.22 and 3.1.23 have not been released, and tags created for these versions have been deleted. I was just testing the new workflow